.gitmodules

```diff
@@ -61,6 +61,7 @@
 [submodule "src/common"]
 	path = src/common
 	url = https://github.com/aspnet/common
+	branch = servicing/1.0.x
```

Comment:
Why the servicing/1.0.x branch?

Reply:
NuGet depends on Microsoft.Extensions.CommandLineUtils 1.0.1. This branch was created from our v1.0.1 tag.